test/specs/parsing/axes/AncestorAxis.tests.ts

```diff
@@ -28,6 +30,26 @@ describe('ancestor', () => {
 		], documentNode);
 		chai.assert.deepEqual(evaluateXPathToStrings('ancestor-or-self::*/position()', documentNode.documentElement.firstChild), ['1', '2']);
 	});
+
+	it('passes buckets for ancestor', () => {
```

Comment:
Comments on this test file also apply on the others. I.e. typing, tslint, explicit tests.